openquake/sep/liquefaction/lateral_spreading.py

```diff
@@ -104,38 +129,64 @@ def hazus_lateral_spreading_displacement_fn(
         )
         a[3.0 < pga_ratio] = 70.0 * pga_ratio[3.0 < pga_ratio] - 180.0
 
-    mag_corr_factor = hazus_disp_mag_correction_factor_spreading(mag)
-    disp_inch = mag_corr_factor * a
+    disp_corr_factor = _hazus_displacement_correction_factor(mag)
+    disp_inch = disp_corr_factor * a
 
     return disp_inch
 
 
-
-def hazus_disp_mag_correction_factor_spreading(
-    mag: float,
-    m3_coeff: float = 0.0086,
-    m2_coeff: float = -0.0914,
-    m1_coeff: float = 0.4698,
-    intercept: float = -0.9835,
-) -> float:
+def lateral_spreading_nonparametric_general(
```

Comment:
Would be good to include the authors' names in the function name, because if and when we expand to include more models, we will need a way to identify the functions corresponding to each model.